stock_analytic/tests/test_stock_picking.py

```diff
@@ -59,7 +56,7 @@ def setUp(self):
         self.outgoing_picking_type = self.env.ref("stock.picking_type_out")
         self.incoming_picking_type = self.env.ref("stock.picking_type_in")
 
-        self.product_categ.update(
+        self.product_categ = self.env.ref("product.product_category_5").copy(
```

Comment:
This is still using demo data as "template", so the side effects are the same: any modification on such demo data will affect here. Please create the product and the category from scratch.

Reply:
Ok sorry, changes done.